_data/simple-icons.json

```diff
@@ -3304,6 +3304,11 @@
             "hex": "F01F7A",
             "source": "https://codecov.io"
         },
+        {
+            "title": "CodeCrafters",
+            "hex": "3475E8",
```

Comment:
```suggestion
            "hex": "171920",
```

This matches what we use in our wordmark, example: 

<img width="414" alt="Screenshot 2024-09-16 at 11 36 04" src="https://github.com/user-attachments/assets/1b81a5a6-7b00-4bab-998f-e900ed5b3b5b">